time-series/time-series-api/src/main/java/com/powsybl/timeseries/TimeSeries.java

```diff
@@ -319,15 +400,14 @@ static void readCsvValues(CsvListReader reader, CsvParsingContext context,
         timeSeriesPerVersion.put(currentVersion, context.createTimeSeries());
     }
 
-    static CsvParsingContext readCsvHeader(CsvListReader csvListReader, String separatorStr) throws IOException {
+    static CsvParsingContext readCsvHeader(CsvListReader csvListReader, String separatorStr, CsvParserConfig csvParserConfig) throws IOException {
```

Comment:
I wonder if the separator should be added in the config and the pass only the config to this method

Reply:
Done